snaps/learn/tutorials/gas-estimation.md

```diff
@@ -211,56 +254,57 @@ To build and test your Snap:
      http://localhost:8000/
    ```
 
-3. Open [`localhost:8000`](http://localhost:8000/) in your browser (with MetaMask Flask installed).
+5.3 Open [`localhost:8000`](http://localhost:8000/) in your browser (with MetaMask Flask installed).
```

Comment:
```suggestion
Open [`localhost:8000`](http://localhost:8000/) in your browser (with MetaMask Flask installed).
```

This would fall under "5.2. Build your Snap."